etc/deltaDebug.py

```diff
@@ -84,16 +93,21 @@ class cutLevel(enum.Enum):
 
 
 class deltaDebugger:
-
     def __init__(self, opt):
         if not os.path.exists(opt.base_db_path):
-            raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT),
-                                    opt.base_db_path)
+            raise FileNotFoundError(
+                errno.ENOENT, os.strerror(errno.ENOENT), opt.base_db_path
+            )
 
         base_db_directory = os.path.dirname(opt.base_db_path)
         base_db_name = os.path.basename(opt.base_db_path)
         self.base_db_file = opt.base_db_path
 
+        self.lib_directory , self.lef_directory = parse_vars_file(parse_vars_file_name(opt.step))
```

Comment:
the name is confusing here. Is it self.lib/lef_directory? If so, how is this going to work when .lib files are spread across multiple directories? What about technology files from the PDK?